### ISSUE TITLE
use Mac as server and Windows 7 as client, not able to type ` this character

### ISSUE THREAD

**Issue by anykx** [debauchee/barrier#1635]
### What happened?

Server: MacBook Pro 2022, latest macOS
Client: Windows 7 SP1

basically, barrier is work fine. just a little problem, when i am using the keyboard on my mac, i am not able to type ` on my client (windows). it doesn't response anything and just this character.

doesn't anyone can advise? much appreciated.

i have tried to search from the past issues, sorry if anyone asked same question.

### Version

v2.4.0

### If applicable, where did you install Barrier from?

download installer from github

### What OSes are you seeing the problem on? (Check all that apply)

Windows

### What OS versions are you using?

Server: MacBook Pro 2022, latest macOS
Client: Windows 7 SP1

**Comment by jroselle** [comment]
Same - Client is Windows 11 latest - neither ~` appear to work

**Comment by giammin** [comment]
this is an old and know issue when using us internationl layout

**Comment by jroselle** [comment]
Do you mean I should switch to US International layout?  On macOS 12.3.1 my keyboard is U.S. (Settings -> Keyboard -> Input Sources.). Same in Windows 11.  I switched them both to US Intl, not working.  So it doesn't work for either US or US International. 

What actually worked for me was creating a hotkey in Barrier (Server config - hotkeys - keystroke (`) and pass that through to the guest.  BUT! Then I lose the ` on the host (macOS.). So it seems I need to choose which pc gets the tick.

**Comment by giammin** [comment]
no, I meant that it seems a synergy known and old bug when using the US International layout:

https://github.com/debauchee/barrier/issues?q=us+international

**Comment by jroselle** [comment]
Gotcha - to be clear it's failing for both Intl and non Intl.  But it seems others are experiencing it too.  When searching the googles, this bug report was the first to show up - apologies, I will move my findings to the other thread.

**Comment by kohend** [comment]
To improve searches, this is called a backtick

**Comment by DMaxter** [comment]
I'm facing the same issue on Linux as server and Windows 10 as client.

I'm using PT keyboard, and I'm able to type on the client ticks and tildes, but not backticks

**Comment by ktonini** [comment]
I too am seeing this issue with Mac as a server and Linux as a client.
